connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResourceTest.java

```diff
@@ -162,7 +171,7 @@ public void testListConnectorsNotLeader() throws Throwable {
 
         PowerMock.replayAll();
 
-        Collection<String> connectors = connectorsResource.listConnectors(FORWARD);
+        Collection<String> connectors = (Collection<String>)connectorsResource.listConnectors(false, forward);
```

Comment:
Would be great to add another unit test for listing connectors (on the non-leader) with expanded results, to serve as a regression test for future changes.